cpp/ql/src/change-notes/2022-01-05-promote-uncontrolled-arithmetic.md

```diff
@@ -0,0 +1,4 @@
+---
+category: newQuery
+---
```

Comment:
I take it the change note process has changed again - is this documented somewhere?

Reply:
Indeed, it has. There's an open PR that documents the process, but it hasn't been merged yet. See here: https://github.com/github/codeql/pull/7400.

Reply:
Ah, thanks!